community/modules/compute/schedmd-slurm-gcp-v5-node-group/variables.tf

```diff
@@ -304,6 +304,17 @@ variable "reservation_name" {
   nullable    = false
 }
 
+variable "maintenance_interval" {
+  description = "Specifies the frequency of planned maintenance events. The accepted values are: PERIODIC"
```

Comment:
I would set `nullable = false` here. Among other reasons, this will generate an error (rather than validation failure):

```
> contains(["", "PERIODIC"], null)
╷
│ Error: Invalid function argument
│
│   on <console-input> line 1:
│   (source code not available)
│
│ Invalid value for "value" parameter: argument must not be null.
╵
```